packages/block-editor/src/components/block-list/block-list-item-cell.native.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+import { View } from 'react-native';
+
+/**
+ * WordPress dependencies
+ */
+import { useEffect, useCallback } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { useBlockListContext } from './block-list-context';
+
+function BlockListItemCell( props ) {
+	const { clientId, rootClientId } = props;
```

Comment:
Nitpick: We could destructure the props directly in the function definition.
```suggestion
function BlockListItemCell( { clientId, rootClientId, children } ) {
```

Reply:
Cool! I'll update it 🙇

Reply:
Updated in https://github.com/WordPress/gutenberg/pull/39089/commits/a09bceec5589b52627d640acc8f56488e3f30890, thanks for the suggestion 👍